src/otx/algo/detection/yolox.py

```diff
@@ -76,13 +76,16 @@ def _exporter(self) -> OTXModelExporter:
             raise ValueError(msg)
 
         swap_rgb = not isinstance(self, YOLOXTINY)  # only YOLOX-TINY uses RGB
+        resize_mode: Literal["standard", "fit_to_window_letterbox"] = "fit_to_window_letterbox"
+        if self.tile_config.enable_tiler:
+            resize_mode = "standard"
```

Comment:
```suggestion
        resize_mode: str = "fit_to_window_letterbox" if self.tile_config.enable_tiler else "standard"
```
And `Literal` seems unnecessary because `resize_mode` is an internal constant in the function.

Reply:
It won't work. The reason for adding literal is for mypy.